node/src/BaseClient.ts

```diff
@@ -3015,6 +3016,36 @@ export class BaseClient {
         );
     }
 
+    /**
+     * Returns the number of members in the sorted set stored at <code>key</code> with scores between
```

Comment:
Using single quotation instead?
```suggestion
     * Returns the number of members in the sorted set stored at `key` with scores between
```